packages/core/src/_test/setup.ts

```diff
@@ -263,7 +264,12 @@ export async function setupAnvil(context: TestContext) {
   );
   context.networks = networks;
   context.requestQueues = requestQueues;
-  context.sources = sources as [LogSource, FactorySource, BlockSource];
+  context.sources = sources as [
+    LogSource,
+    FactorySource,
+    CallTraceSource,
```

Comment:
The ordering of the sources changed which is gonna show up in the test files diff